kibble/api/films_test.go

```diff
@@ -120,6 +120,9 @@ func getFilm() filmV2 {
 		SeoTitle:       "Film One Meta Title",
 		SeoKeywords:    "Film One Meta Keywords",
 		SeoDescription: "Film One Meta Description",
+		Refs: models.CustomFields{
```

Comment:
Even though it has the same shape, I think it's worth a separate model to not have the question later of "why are custom fields being used in refs?".
```suggestion
		Refs: models.Refs{
```